### PR TITLE
Patch/tests + added yarn test command to deploy.yml

### PR DESCRIPTION
**Pull request by antiyro**
Patched old not working tests to current version. Those tests may be outdated because they don't correspond to Pangolin v2 (they were initialy made for uniswap and never been touched since Pangolin). Also modified deploy.yml so CI is good.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - master
       - dev
+      - patch/tests
```
Comment (iam-robi) [inline .github/workflows/deploy.yml:8]: we can keep only for mast / dev for now

Comment (antiyro) [reply .github/workflows/deploy.yml]: Just corrected it

**Review by HariSeldon23 — APPROVED** [review]

**Comment by HariSeldon23** [comment]
Looks good, happy to merge whenever everybody is ready

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]
@antiyro tests are looking good but we dont need yarn.lock file changes as there is no package.json file change. so please revert yarn.lock file then we are good to merge

**Review by SarjuHansaliya — APPROVED** [review]

**Pull request merged**